src/components/layout/theme.tsx

```diff
@@ -2,7 +2,8 @@ import { DefaultTheme } from 'styled-components';
 
 export const HEADER_HEIGHT_VALUE = 65;
 export const CONTENT_SPACE_SMALL = 80;
-export const CONTENT_SPACE_LARGE = 180;
+export const CONTENT_SPACE_LARGE = 120;
+export const CONTENT_SPACE_LEFT_RIGHT = 76;
```

Comment:
I think it makes more sense to remove the left and right padding (= `CONTENT_SPACE_LEFT_RIGHT`) and use the maximum content width instead of the 1400px. According to Figma this is currently 1184px. Since the value seems a bit random to me, I would definitely clarify this again with the design team.

Reply:
I though the box which is selected on the screen is our content-block-container

<img width="1004" alt="image" src="https://user-images.githubusercontent.com/3524642/222213782-f6fa42a6-bce4-4668-a322-ebe66e510fa7.png">

then I found that child element of this container (text-wrapper) is 1248

<img width="988" alt="image" src="https://user-images.githubusercontent.com/3524642/222214414-23ea6685-20df-4496-b818-4d52d3d03e18.png">

then I did simple calculation (1400-1248)/2 = 76px to find the padding value 
Now I understand that all these calculations were made with the originally wrong approach :)

And yes, let us clarify and do it in a right way

Reply:
I though the box which is selected on the screen is our content-block-container
<img width="852" alt="image" src="https://user-images.githubusercontent.com/3524642/222220903-1ea23670-89ce-4e0e-a665-98dba5fc1b32.png">
And it is supposed to be 108 px not 76px. I came to 76 using calculation with the originally wrong approach :)

But anyway, let us clarify and do it in a right way.

Reply:
I learned from the design team that the container's width should actually be 1400px and the content inside is 1184px. Since there is no need for a container (and it doesn't exist in the project), I set the content-block size at 1184px and removed the padding.